tests/javascript/webpack.test.config.js

```diff
@@ -39,13 +39,19 @@ module.exports = {
             {
                 test: /\.tsx?$/,
```

Comment:
While I agree that all React components will end in .tsx, there could be supporting files that end in .ts that would also be good to unit test. Should this be updated to /\.(ts|tsx)?$/, ?

Reply:
That's what the ? mark at the end is for. The x is an optional character.

Reply:
d'oh.... regex fail on my end.